### ISSUE TITLE
no dupes shown in web interface but in CLI

### ISSUE THREAD

**Issue by j-lakeman** [eldertek/duplicatefinder#26]
`occ duplicates:list` gives me a list of all duplicates as expected.
`occ duplicates:list --user USER` takes very long compared to the above command.
The web interface shows no duplicates at all.

**Comment by eldertek** [comment]
What version are you ? 
 
The app worked before ?

**Comment by j-lakeman** [comment]
Nextcloud Hub 6 (27.1.3)
Duplicate Finder 1.1.0

**Comment by eldertek** [comment]
Any logs ?

**Comment by j-lakeman** [comment]
Hope this is sufficient:


```
{"reqId":"dwrc76bUTAVDrg8iEBn4",
"level":3,
"time":"2023-11-05T21:55:47+00:00",
"remoteAddr":"",
"user":"--",
"app":"duplicatefinder",
"method":"",
"url":"--",
"message":"Failed to handle NewFileInfoEvent.",
"userAgent":"--",
"version":"27.1.3.2",
"exception":{"Exception":"OC\\User\\NoUserException",
"Message":"Backends provided no user object",
"Code":0,
"Trace":[{"function":"getUserFolder",
"class":"OC\\Files\\Node\\Root",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/Files/Node/LazyFolder.php",
"line":74,
"function":"call_user_func_array"},
{"file":"/var/www/nextcloud/lib/private/Files/Node/LazyRoot.php",
"line":40,
"function":"__call",
"class":"OC\\Files\\Node\\LazyFolder",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Service/FolderService.php",
"line":40,
"function":"getUserFolder",
"class":"OC\\Files\\Node\\LazyRoot",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Service/FileInfoService.php",
"line":211,
"function":"getNodeByFileInfo",
"class":"OCA\\DuplicateFinder\\Service\\FolderService",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Listener/FileInfoListener.php",
"line":43,
"function":"calculateHashes",
"class":"OCA\\DuplicateFinder\\Service\\FileInfoService",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/EventDispatcher/ServiceEventListener.php",
"line":86,
"function":"handle",
"class":"OCA\\DuplicateFinder\\Listener\\FileInfoListener",
"type":"->"},
{"file":"/var/www/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php",
"line":251,
"function":"__invoke",
"class":"OC\\EventDispatcher\\ServiceEventListener",
"type":"->"},
{"file":"/var/www/nextcloud/3rdparty/symfony/event-dispatcher/EventDispatcher.php",
"line":73,
"function":"callListeners",
"class":"Symfony\\Component\\EventDispatcher\\EventDispatcher",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/EventDispatcher/EventDispatcher.php",
"line":94,
"function":"dispatch",
"class":"Symfony\\Component\\EventDispatcher\\EventDispatcher",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/EventDispatcher/EventDispatcher.php",
"line":106,
"function":"dispatch",
"class":"OC\\EventDispatcher\\EventDispatcher",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Service/FileInfoService.php",
"line":151,
"function":"dispatchTyped",
"class":"OC\\EventDispatcher\\EventDispatcher",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Utils/ScannerUtil.php",
"line":90,
"function":"save",
"class":"OCA\\DuplicateFinder\\Service\\FileInfoService",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Utils/ScannerUtil.php",
"line":79,
"function":"saveScannedFile",
"class":"OCA\\DuplicateFinder\\Utils\\ScannerUtil",
"type":"->"},
{"function":"OCA\\DuplicateFinder\\Utils\\{closure}",
"class":"OCA\\DuplicateFinder\\Utils\\ScannerUtil",
"type":"->",
"args":["*** sensitive parameters replaced ***"]},
{"file":"/var/www/nextcloud/lib/private/Hooks/EmitterTrait.php",
"line":105,
"function":"call_user_func_array"},
{"file":"/var/www/nextcloud/lib/private/Hooks/PublicEmitter.php",
"line":40,
"function":"emit",
"class":"OC\\Hooks\\BasicEmitter",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/Files/Utils/Scanner.php",
"line":141,
"function":"emit",
"class":"OC\\Hooks\\PublicEmitter",
"type":"->"},
{"function":"OC\\Files\\Utils\\{closure}",
"class":"OC\\Files\\Utils\\Scanner",
"type":"->",
"args":["*** sensitive parameters replaced ***"]},
{"file":"/var/www/nextcloud/lib/private/Hooks/EmitterTrait.php",
"line":105,
"function":"call_user_func_array"},
{"file":"/var/www/nextcloud/lib/private/Files/Cache/Scanner.php",
"line":244,
"function":"emit",
"class":"OC\\Hooks\\BasicEmitter",
"type":"->"},
{"file":"/var/www/nextcloud/apps/files_sharing/lib/External/Scanner.php",
"line":57,
"function":"scanFile",
"class":"OC\\Files\\Cache\\Scanner",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/Files/Cache/Scanner.php",
"line":460,
"function":"scanFile",
"class":"OCA\\Files_Sharing\\External\\Scanner",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/Files/Cache/Scanner.php",
"line":397,
"function":"handleChildren",
"class":"OC\\Files\\Cache\\Scanner",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/Files/Cache/Scanner.php",
"line":400,
"function":"scanChildren",
"class":"OC\\Files\\Cache\\Scanner",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/Files/Cache/Scanner.php",
"line":347,
"function":"scanChildren",
"class":"OC\\Files\\Cache\\Scanner",
"type":"->"},
{"file":"/var/www/nextcloud/apps/files_sharing/lib/External/Scanner.php",
"line":39,
"function":"scan",
"class":"OC\\Files\\Cache\\Scanner",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/Files/Utils/Scanner.php",
"line":269,
"function":"scan",
"class":"OCA\\Files_Sharing\\External\\Scanner",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Utils/ScannerUtil.php",
"line":64,
"function":"scan",
"class":"OC\\Files\\Utils\\Scanner",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Service/FileInfoService.php",
"line":263,
"function":"scan",
"class":"OCA\\DuplicateFinder\\Utils\\ScannerUtil",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Command/FindDuplicates.php",
"line":171,
"function":"scanFiles",
"class":"OCA\\DuplicateFinder\\Service\\FileInfoService",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Command/FindDuplicates.php",
"line":152,
"function":"findDuplicates",
"class":"OCA\\DuplicateFinder\\Command\\FindDuplicates",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/User/Manager.php",
"line":527,
"function":"OCA\\DuplicateFinder\\Command\\{closure}",
"class":"OCA\\DuplicateFinder\\Command\\FindDuplicates",
"type":"->",
"args":["*** sensitive parameters replaced ***"]},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Command/FindDuplicates.php",
"line":153,
"function":"callForAllUsers",
"class":"OC\\User\\Manager",
"type":"->"},
{"file":"/var/www/nextcloud/apps/duplicatefinder/lib/Command/FindDuplicates.php",
"line":114,
"function":"findAllDuplicates",
"class":"OCA\\DuplicateFinder\\Command\\FindDuplicates",
"type":"->"},
{"file":"/var/www/nextcloud/3rdparty/symfony/console/Command/Command.php",
"line":298,
"function":"execute",
"class":"OCA\\DuplicateFinder\\Command\\FindDuplicates",
"type":"->"},
{"file":"/var/www/nextcloud/3rdparty/symfony/console/Application.php",
"line":1040,
"function":"run",
"class":"Symfony\\Component\\Console\\Command\\Command",
"type":"->"},
{"file":"/var/www/nextcloud/3rdparty/symfony/console/Application.php",
"line":301,
"function":"doRunCommand",
"class":"Symfony\\Component\\Console\\Application",
"type":"->"},
{"file":"/var/www/nextcloud/3rdparty/symfony/console/Application.php",
"line":171,
"function":"doRun",
"class":"Symfony\\Component\\Console\\Application",
"type":"->"},
{"file":"/var/www/nextcloud/lib/private/Console/Application.php",
"line":211,
"function":"run",
"class":"Symfony\\Component\\Console\\Application",
"type":"->"},
{"file":"/var/www/nextcloud/console.php",
"line":100,
"function":"run",
"class":"OC\\Console\\Application",
"type":"->"},
{"file":"/var/www/nextcloud/occ",
"line":11,
"args":["/var/www/nextcloud/console.php"],
"function":"require_once"}],
"File":"/var/www/nextcloud/lib/private/Files/Node/Root.php",
"Line":364,
"message":"Failed to handle NewFileInfoEvent.",
"exception":{},
"CustomMessage":"Failed to handle NewFileInfoEvent."}}
```

**Comment by j-lakeman** [comment]
I reset the duplicates list and ran a manual scan on my user and got this:

![image](https://github.com/eldertek/duplicatefinder/assets/101660243/764f7640-2a45-4b92-9c7a-64c972d658e3)

**Comment by eldertek** [comment]
Have you tried uninstalling the reinstalling the app ?

**Comment by j-lakeman** [comment]
Same same

**Comment by j-lakeman** [comment]
It does show the found duplicates in json format

**Comment by j-lakeman** [comment]
Dont have it, sorry.
Could you pls write me an e-mail to le0324tb@anonaddy.me?

**Comment by eldertek** [comment]
Can you inspect web element and go on duplicates finder page, then give log of the console of the browser also check from network if the url I asked is rightly fetched.

**Comment by j-lakeman** [comment]
Resolved with new version!
Cheers!

**Comment by remz1337** [comment]
Hey sorry to reopen this but I'm having the exact same issue. Right after install it found some duplicates in my shared admin user folder (folder shared with other users). But then the web UI is saying no duplicates found whilst I can see duplicates for that user when using the command list all.

Accessing the api/duplicates/all URL, I see the results in the body of the page but console is showing some errors:
![image](https://github.com/eldertek/duplicatefinder/assets/9980037/42b69670-9929-471e-b587-14308f549287)

**Comment by whitecloudy** [comment]
I had the same issue like this. All the duplicates can be find through CLI but not Web. All I could get was messages saying "Could not fetch acknowledged duplicates" and "Could not fetch unacknowledged duplicates".

First of all, my MariaDB was clogged because of some upgrade issue. I fixed it by following [this](https://github.com/go-gitea/gitea/issues/25970). 

I'm not sure this MariaDB issue makes this issue but it did not changed instantly. So I just cleared the duplicates information and ordered re-scan right now. I will tell you if this works.
